std_msgs/Header header
pf_trunk_width/TreeInfo[] trees